src/components/create/NumberOfAttempts.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
Let's keep that for another PR?

Reply:
Yes, I created the migration branch on the NumberOfAttempts instead of creating new branch from main...